devise_2fa.gemspec

```diff
@@ -18,12 +18,12 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
-  gem.add_dependency "rails", "~> 5.0"
+  gem.add_dependency "rails", ">= 4.1", "< 6.1"
```

Comment:
Would you be willing to lift the rails ceiling? I find more often than not a dependency like this can hold back a minor rails release more often than it saves the user from a breaking change in a future rails release (which can typically be reported/debugged fairly easily).

Reply:
I do believe @bettinson said that there was something in lib/ that was reliant on 5.0+, but I don't know what feature he saw was utilized.
Mentioning him in case he knows what feature was 5.0+

Reply:
I don't specifically recall what was reliant on 5.0, but I remember it being multiple things. Does the test suite pass with this change @williamatodd ?

Reply:
The test suite you set up doesn't use anything like Appraisal to know for certain @bettinson

Reply:
Before lifting the ceiling beyond 6.1, Appraisal should probably be implemented.  At the very least, I'd like to avoid throwing scary deprecation warnings in the test suite.

Reply:
Not a bad idea

Reply:
If there's deprecation warnings it's probably worth fixing them.
Is this something you'd like help on from the @rietta-inc team?

Reply:
@grepsedawk you are more than welcome to contribute fixes as needed.  I maintain this gem because I use it in production, but I don't really devote a lot of energy to fixing things that aren't actually broken.